internal/events/event.go

```diff
@@ -82,6 +86,23 @@ func (e *Events) v2SaveEvent(ctx context.Context, clusterID *strfmt.UUID, hostID
 			tx.Rollback()
 		} else {
 			tx.Commit()
+			key := ""
+			if clusterID != nil {
+				key = clusterID.String()
+			}
+			log.WithFields(logrus.Fields{
+				"event": event,
+			}).Debugf("sending event")
+			err := eventstream.StreamEvent(
+				ctx,
+				e.stream,
+				"Event",
+				key,
+				event.Event,
+			)
+			if err != nil {
```

Comment:
what will happen if there is no connection to the Kafka instance? Does it mean that all the service logs will be full of those error messages?